src/helpers/database/index.js

```diff
@@ -6,7 +6,8 @@ module.exports = async () => {
   try {
     await mongoose.connect(__config.mongodb.url);
     await logger.info('Connected to the database');
-  } catch (err) {
+  }
+ catch (err) {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 1 tab but found 1 space.](https://app.codacy.com/gh/ZynerOrg/xyter/pullRequest?prid=9008883)
```suggestion
	catch (err) {
```